Formula/p/purr.rb

```diff
@@ -0,0 +1,30 @@
+class Purr < Formula
+  desc "Versatile zsh CLI tool for viewing and searching through Android logcat output"
+  homepage "https://github.com/google/purr"
+  url "https://github.com/google/purr/archive/refs/tags/2.0.0.tar.gz"
+  sha256 "f4e6c3dcdd386da7d3664fe6c947e9cfdb2e3362ca8c78e47d0445d94d471246"
+  license "Apache-2.0"
+  head "https://github.com/google/purr.git", branch: "main"
+
+  depends_on "make" => :build
+  depends_on "fzf"
+  depends_on "zsh"
```

Comment:
If there is not a need for some bleeding-edge `zsh` features, I would suggest `uses_from_macos "zsh"` instead of `depends_on "zsh"`.

This allows depending on the macOS-provided copy of `zsh` on macOS, which may not necessarily be the newest (currently 5.8.1 on my Monterey machine, which is the oldest macOSversion that we package for at the moment - possibly newer `zsh` on newer macOS versions). On Linux, the `uses_from_macos` will still cause Homebrew `zsh` to be installed, same behavior as `depends_on "zsh"`.